From 72e0d7d1d8d93d1becf1c2e1199f72a9529adad5 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 27 Apr 2018 11:54:16 +0800
Subject: [PATCH] arm64: rk3308_linux_defconfig: enable
 HARDLOCKUP_PANIC/PANIC_ON_OOPS and PANIC_TIMEOUT=1

Make sure kernel panic and reboot when something wrong.

Change-Id: Iaed051431fa4ae2fb5bdd952737deb07a24a33c1
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rk3308_linux_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/rk3308_linux_defconfig b/arch/arm64/configs/rk3308_linux_defconfig
index 9d6e13f147e4..0b13c80d01d2 100644
--- a/arch/arm64/configs/rk3308_linux_defconfig
+++ b/arch/arm64/configs/rk3308_linux_defconfig
@@ -242,8 +242,11 @@ CONFIG_DEBUG_FS=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_MAGIC_SYSRQ_DEFAULT_ENABLE=0
 CONFIG_LOCKUP_DETECTOR=y
+CONFIG_BOOTPARAM_HARDLOCKUP_PANIC=y
 CONFIG_BOOTPARAM_SOFTLOCKUP_PANIC=y
 CONFIG_BOOTPARAM_HUNG_TASK_PANIC=y
+CONFIG_PANIC_ON_OOPS=y
+CONFIG_PANIC_TIMEOUT=1
 # CONFIG_SCHED_DEBUG is not set
 # CONFIG_DEBUG_PREEMPT is not set
 CONFIG_RCU_CPU_STALL_TIMEOUT=60
-- 
2.35.3

